packages/react/src/components/Table/Table.tsx

```diff
@@ -0,0 +1,57 @@
+import { ReactElement, CSSProperties } from 'react';
+import '@compassion-gds/css/src/components/Table/table.css';
+
+export interface ColumnCell {
```

Comment:
Let's rename this to `Column` - it's about the entire column and the (column) key used by each row in for that column